### PR TITLE
Add elyra-r-editor-extension package

### PR DESCRIPTION
**Pull request by akchinSTC**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by akchinSTC** [comment]
@lresende, need an approve as maintainer here =)

**Review by lresende — APPROVED** [review]

File: recipes/elyra-r-editor-extension/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    - elyra-server ==2.2.2
+    - python >=3.6
+
+test:
+  commands:
+    - exit 0
+
+about:
+  home: https://github.com/elyra-ai/elyra
+  summary: Elyra provides AI Centric extensions to JupyterLab
```
Comment (ptitzler) [inline recipes/elyra-r-editor-extension/meta.yaml:32]:
```suggestion
  summary: Elyra JupyterLab extension providing support for R scripts. Learn more at https://elyra.readthedocs.io/en/stable/user_guide/enhanced-script-support.html
```

**Comment by akchinSTC** [comment]
@marcelotrevisani -  any chance you could help us approve this? Luciano gave an approval so we should be good =)

File: LICENSE.txt [hunk 2]
```diff
@@ -1,27 +1,13 @@
-Copyright (c) 2015-2021, conda-forge
+BSD 3-clause license
+Copyright (c) 2015-2020, conda-forge contributors
```
Comment (marcelotrevisani) [inline LICENSE.txt:2]: Could you please revert this file here?

Comment (akchinSTC) [reply LICENSE.txt:2]: woop, can do!

File: LICENSE.txt [hunk 3]
```diff
@@ -24,4 +24,4 @@ DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
 SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
 CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
 OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
-OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```
Comment (marcelotrevisani) [inline LICENSE.txt:27]: can you revert it please? just to have a clean PR with modifications just related to the recipe

File: recipes/elyra-r-editor-extension/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - elyra-server ==2.2.2
+    - python >=3.6
+
+test:
+  commands:
+    - exit 0
```
Comment (marcelotrevisani) [inline recipes/elyra-r-editor-extension/meta.yaml:28]: Could you please really test the software here?

Comment (marcelotrevisani) [inline recipes/elyra-r-editor-extension/meta.yaml:28]: maybe
```suggestion
  imports:
    - elyra_r_editor_extension
  commands:
    - pip check
  requires:
    - pip
```

**Pull request closed without merging**